DFC.ServiceTaxonomy.PageLocation/Handlers/PageLocationPartHandler.cs

```diff
@@ -1,16 +1,56 @@
-﻿using System.Threading.Tasks;
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
 using DFC.ServiceTaxonomy.PageLocation.Models;
+using DFC.ServiceTaxonomy.PageLocation.Services;
 using OrchardCore.ContentManagement;
 using OrchardCore.ContentManagement.Handlers;
+using OrchardCore.ContentManagement.Records;
+using OrchardCore.Title.Models;
+using YesSql;
 
 namespace DFC.ServiceTaxonomy.PageLocation.Handlers
 {
     public class PageLocationPartHandler : ContentPartHandler<PageLocationPart>
     {
+        private readonly ISession _session;
+        private readonly IPageLocationClonePropertyGenerator _generator;
+
+        public PageLocationPartHandler(ISession session, IPageLocationClonePropertyGenerator generator)
+        {
+            _session = session;
+            _generator = generator;
+        }
+
         public override Task InitializingAsync(InitializingContentContext context, PageLocationPart part)
         {
             part.Apply();
             return Task.CompletedTask;
         }
+
+        //public override async Task CloningAsync(CloneContentContext context, PageLocationPart part)
```

Comment:
Might as well delete this now - it's in git's history if we need to resurrect it.